src/main/java/com/group12/LaunchInterceptorConditionCollection.java

```diff
@@ -68,6 +68,38 @@ public boolean LIC1(List<Point> points, double radius1) throws IllegalArgumentEx
         return false;
     }
 
+    /**
+     * LIC #3 checks if there exists at least one set of three consecutive data points that are the vertices of a triangle
+     * with an area greater than <b>area1</b>.
+     *
+     * @param points list of radar echos ({@link Point})
+     * @param area1 has to be greater than or equal to 0 (zero)
+     * @return true if at least one set of three consecutive data points forms a triangle with an area greater than
+     * given <b>area1</b>, false otherwise.
+     * @throws IllegalArgumentException is thrown if <b>area1</b> is less than 0 (zero)
+     */
+    public boolean LIC3(List<Point> points, double area1) throws IllegalArgumentException {
+        if (points == null) {
+            throw new IllegalArgumentException("Points list cannot be null");
+        }
+        if (points.size() < 3) {
+            throw new IllegalArgumentException("There has to be at least three points, size of given points="
+                    + points.size());
+        }
+        if (area1 < 0) {
+            throw new IllegalArgumentException("Area cannot be less than zero");
+        }
+        for (int i = 0; i < points.size() - 2; i++) {
+            Point point1 = points.get(i);
+            Point point2 = points.get(i + 1);
+            Point point3 = points.get(i + 2);
+            if (areaOfTriangle(point1, point2, point3) > area1) {
```

Comment:
Here we should probably use the doubleCompare-function instead of normal double comparison.

Reply:
Yes, will fix.